Sources/AuthFoundation/JWT/Claim.swift

```diff
@@ -280,7 +280,7 @@ public extension HasClaims {
     
     var allClaims: [String] {
         Array([
-            claims.map({ $0.rawValue }),
+            claims.map(\.rawValue),
```

Comment:
What does this change do?

Reply:
It's using a Swift short-hand to eliminate the need for a closure. Just to tidy up the source code.